hooks/pre/34-pulpcore_directory_layout.rb

```diff
@@ -0,0 +1,15 @@
+unless app_value(:noop)
+  require 'pathname'
+
+  PULP_ROOT = Pathname.new('/var/lib/pulp')
+  LEGACY_DIR = PULP_ROOT / 'artifact'
+  NEW_MEDIA_ROOT = PULP_ROOT / 'media'
+  DESTINATION = NEW_MEDIA_ROOT / LEGACY_DIR.basename
```

Comment:
Could we store these directories in a single place for reference?

Reply:
At this point I think it would only add confusion because you need to look at other files to see what's going on.